lib/resources/host.rb

```diff
@@ -113,6 +113,7 @@ def ipaddress
     end
 
     def to_s
+      return "Host #{hostname} port #{port} proto #{protocol}" if port
```

Comment:
I think this is fine. I'd consider "building" the string instead of the early `return` so we're not duplicating the "Host hostname" bit... something like this:

```ruby
def to_s
  resource_name = "Host #{hostname}"
  resource_name += " port #{port} proto #{protocol}" if port

  resource_name
end
```

Reply:
Sounds good, just pushed out a update with the changes. Thanks for the help!